### PR TITLE
Standard omg hashes

### PR DESCRIPTION
**Pull request by heaptracetechnology**
Updated README file format and copyright in License file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,39 @@
...
+[![Open Microservice Guide](https://img.shields.io/badge/OMG%20Enabled-👍-green.svg?)](https://microservice.guide)
+<!-- [![Build status](https://img.shields.io/travis/microservice/hashes/master.svg?style=for-the-badge)](https://travis-ci.org/microservice/hashes) -->
 
 This OMG service provides various digest and hashing capabilities.
 
-Usage
------
+## Direct usage in [Storyscript](https://storyscript.io/):
 
+##### Digest
```
Comment (wilzbach) [inline README.md]: whitespace

File: README.md [hunk 2]
```diff
@@ -1,21 +1,39 @@
...
 ```coffee
-# Storyscript
 digest = hashes digest method: "sha1" data: "hello world"
 # {"method":"sha1","digest":"2AAE6C35C94FCFB415DBE95F408B9CE91EE846ED"}
 ```
-
+##### Hmac
```
Comment (wilzbach) [inline README.md]: whitespace

File: README.md [hunk 3]
```diff
@@ -1,21 +1,39 @@
-Hashes
-=======
+# _Hashes_ OMG Microservice
 
-[![Build status](https://img.shields.io/travis/microservice/hashes/master.svg?style=for-the-badge)](https://travis-ci.org/microservice/hashes)
+[![Open Microservice Guide](https://img.shields.io/badge/OMG%20Enabled-👍-green.svg?)](https://microservice.guide)
+<!-- [![Build status](https://img.shields.io/travis/microservice/hashes/master.svg?style=for-the-badge)](https://travis-ci.org/microservice/hashes) -->
```
Comment (wilzbach) [inline README.md]: Why not fix the error?

https://travis-ci.org/omg-services/hashes

Looks like a regression with OMG.

Comment (rohit121) [reply README.md]: @ht68rohit can you take a look ?

Comment (heaptracetechnology) [reply README.md]: @rohit121 Yes, I have updated Travis badge

**Pull request merged**